src/xcode/ENA/ENA/Source/Services/CoronaTestService/FamilyMemberCoronaTestService.swift

```diff
@@ -788,7 +791,6 @@ class FamilyMemberCoronaTestService: FamilyMemberCoronaTestServiceProviding {
 					isNew: false,
 					testResult: testResult,
 					finalTestResultReceivedDate: testResult == .pending ? nil : Date(),
-					testResultWasShown: LaunchArguments.familyMemberTest.pcr.positiveTestResultWasShown.boolValue,
```

Comment:
Do we use the launch arguments (this one and the one for antigen) anywhere? I think we should either replace them with new ones for the new property or remove them completely